src/Testcontainers/Builders/TestcontainersBuilderCosmosDbExtension.cs

```diff
@@ -0,0 +1,29 @@
+namespace DotNet.Testcontainers.Builders
+{
+  using System.Linq;
+  using DotNet.Testcontainers.Configurations;
+  using DotNet.Testcontainers.Containers;
+  using JetBrains.Annotations;
+
+  [PublicAPI]
+  public static class TestcontainersBuilderCosmosDbExtension
```

Comment:
All other classes inherit from `TestcontainerDatabase` (or equivalent), why do we not use `TestcontainersBuilderDatabaseExtension`?

Reply:
Yes, I agree. Had to set the database to something. Maybe this should be a constructor parameter?